Survey/Survey/Sources/Presentation/Modules/Splash/SplashView.swift

```diff
@@ -9,10 +9,26 @@ import SwiftUI
 
 struct SplashView: View {
 
+    @State private var logoOpacity = 0.0
+
     var body: some View {
         ZStack {
-            Color(.systemTeal)
-            Text("Survey App")
+            Image(Images.splashBackground)
+                .resizable()
+                .aspectRatio(contentMode: .fill)
+                .edgesIgnoringSafeArea(.top)
+            Image(Images.overlay)
+                .resizable()
+                .aspectRatio(contentMode: .fill)
+                .edgesIgnoringSafeArea(.top)
```

Comment:
Since we have more than one screen using overlay (with a little different color opacity), what do you think if we recreate the __overlay__ image with a custom SwiftUI view? Then we don't have to import many assets for __overlay__ images from Figma, like `overlay1`, `overlay2`, ... If so, you should have a look at [LinearGradient](https://developer.apple.com/documentation/swiftui/lineargradient) 🙏

Reply:
updated in this [commit](https://github.com/Shayokh144/ic-survey-swiftui-combine/pull/38/commits/ed946222cb171779b44bd26dcc147c0fbc9ed649)